tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -3,22 +3,27 @@
 import pytest
 
 from ocs_ci.framework.testlib import ocs_upgrade
-from ocs_ci.ocs.ocs_upgrade import run_ocs_upgrade, tier4_functions_to_run
+from ocs_ci.ocs.ocs_upgrade import run_ocs_upgrade, UpgradeDisruptiveOps
 
 
 log = logging.getLogger(__name__)
 
-
-@pytest.mark.polarion_id("OCS-1579")
-def test_worker_node_abrupt_shutdown(nodes):
-    run_ocs_upgrade(nodes, dictionary_name=tier4_functions_to_run,
-                    function_runner="worker_abrupt_shutdown")
-
-
-@pytest.mark.polarion_id("OCS-1575")
-def test_worker_node_permanent_shutdown(nodes):
-    run_ocs_upgrade(nodes, dictionary_name=tier4_functions_to_run,
-                    function_runner="worker_perm_shutdown")
+dist = UpgradeDisruptiveOps()
+
+
+@pytest.mark.parametrize(
+    argnames=["operation", "operation_args"], argvalues=[
+        pytest.param(
+            [dist.worker_node_shutdown], [True], marks=pytest.mark.polarion_id("OCS-1575")
```

Comment:
I think it should be `[dist.worker_node_shutdown, True]`

Reply:
it could be as it is now, or this way: 
```@pytest.mark.parametrize(
    argnames=["operation", "operation_args"], argvalues=[
        pytest.param(
            dist.worker_node_shutdown, True, marks=pytest.mark.polarion_id("OCS-1575")
        ),
        pytest.param(
            dist.worker_node_shutdown, False, marks=pytest.mark.polarion_id("OCS-1579")
        ),
    ]
)
```
i will fix it